specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/examples/SecurityCenterServiceList.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "api-version": "2021-01-11",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.SecurityAndCompliance/privateLinkServicesForM365SecurityCenter/service1",
+            "name": "service1",
+            "location": "West US",
+            "type": "Microsoft.SecurityAndCompliance/privateLinkServicesForM365SecurityCenter",
+            "kind": "fhir-R4",
+            "etag": "etag",
+            "tags": {},
+            "properties": {
+              "provisioningState": "Creating",
+              "accessPolicies": [
+                {
+                  "objectId": "c487e7d1-3210-41a3-8ccc-e9372b78da47"
+                },
+                {
+                  "objectId": "5b307da8-43d4-492b-8b66-b0294ade872f"
+                }
+              ],
+              "cosmosDbConfiguration": {
+                "offerThroughput": 1000,
+                "keyVaultKeyUri": "https://my-vault.vault.azure.net/keys/my-key"
+              },
+              "authenticationConfiguration": {
+                "authority": "https://login.microsoftonline.com/abfde7b2-df0f-47e6-aabf-2462b07508dc",
+                "audience": "https://azurehealthcareapis.com",
+                "smartProxyEnabled": true
+              },
+              "corsConfiguration": {
+                "origins": [
+                  "*"
+                ],
+                "headers": [
+                  "*"
+                ],
+                "methods": [
+                  "DELETE",
+                  "GET",
+                  "OPTIONS",
+                  "PATCH",
+                  "POST",
+                  "PUT"
+                ],
+                "maxAge": 1440,
+                "allowCredentials": false
+              },
+              "privateEndpointConnections": [],
+              "publicNetworkAccess": "Disabled"
+            }
+          }
+        ],
+        "nextLink": "https://host/subscriptions/subid/providers/Microsoft.SecurityAndCompliance/services?api-version=2018-08-20-preview&%24skipToken=e30%3d"
```

Comment:
>services [](start = 96, length = 8)

are you sure this is the correct url path? I don't see you exposing a services type

Reply:
corrected.